Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityCameraSettingsProfile.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.CameraSystem;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality
+{
+    /// <summary>
+    /// Configuration profile for the Windows Mixed Reality Camera settings provider.
+    /// </summary>
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Profiles/Windows Mixed Reality Camera Settings Profile", fileName = "DefaultWindowsMixedRealityCameraSettingsProfile", order = 100)]
+    [MixedRealityServiceProfile(typeof(WindowsMixedRealityCameraSettings))]
+    public class WindowsMixedRealityCameraSettingsProfile : BaseCameraSettingsProfile
+    {
+        [SerializeField]
+        [Tooltip("Whether to use PV camera rendering for Mixed Reality Capture on Windows.")]
```

Comment:
Do customers know what PV means? should we phrase as "color camera" or something?

Reply:
Hmm, good point. It's referred to as PVCamera in most of our platform docs, but I can expand it to PhotoVideoCamera

Reply:
I also added a documentation button next to the setting for more information:

![image](https://user-images.githubusercontent.com/3580640/69374801-bebf8080-0c5b-11ea-87cc-2d800c9ad47a.png)